### PR TITLE
Display all Articles With a Specific Tag

### PR DESCRIPTION
**Pull request by Hadeneekeh**
#### What does this PR do?
- display articles with a tag

#### Description of Task to be completed?
- Renders the articles when a tag is clicked

#### How should this be manually tested?
- clone the repo
- run `npm install`
- checkout to branch `ft/167190403-articles-from-tag`
#### Any background context you want to provide?
- [E2E](https://drive.google.com/file/d/172f8sgmFV63hrA1clcnPZ5PigbVpYdIc/view?usp=sharing)

#### What are the relevant pivotal tracker stories?
[167190403](https://www.pivotaltracker.com/n/projects/2364267/stories/167190403)

### CODE REVIEW

File: src/actions/searchActions.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const getFilteredArticles = () => (dispatch, getState) => {
     }
   });
 
-  return axios.post(`https://a-haven-staging.herokuapp.com/api/v1/articles/search/filter?searchQuery=${searchQuery}&limit=2000`, object)
+  return axios.post(`http://a-haven-staging.herokuapp.com/api/v1/articles/search/filter?searchQuery=${searchQuery}&limit=2000`, object)
```
Comment (Lundii) [inline src/actions/searchActions.js]: Nice work Kafi. I observed you changed `https` to `http`. Probably an oversight. Please look into it.

Comment (Hadeneekeh) [reply src/actions/searchActions.js]: Thanks for the observation. I have made a correction.

**Review by Lundii — APPROVED** [review]

File: src/views/Articles/ReadArticle/ReadArticle.scss [hunk 2]
```diff
@@ -196,3 +196,8 @@
     align-items: center;
   }
 }
+
+.liTag :hover {
+  cursor: pointer;
+  transform: scale(1.2);
+}
```
Comment (obayomi96) [inline src/views/Articles/ReadArticle/ReadArticle.scss]: Please add a new linke here @Hadeneekeh

File: src/views/ArticlesTag/ArticlesTag.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  flex-direction: row;
+}
+
+.card {
+  margin: 30px 30px 30px 0;
+}
+
+h3 {
+  padding: 0px 15px 15px 0px;
+}
```
Comment (obayomi96) [inline src/views/ArticlesTag/ArticlesTag.scss]: And here

**Review by cvjude — APPROVED** [review]
LGTM

**Review by encodedBicoding — APPROVED** [review]
LGTM

**Review by obayomi96 — APPROVED** [review]
LGTM

**Review by chuxmykel — APPROVED** [review]
Nice work @Hadeneekeh. Looks good to me.

**Pull request merged**